Block.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';

```

Comment:
no need for importing React as of v17. You can just create function with capital letter and it will convert jsx automatically